local-helm/buyingcatalogue/charts/isapi/templates/deployment.yaml

```diff
@@ -115,6 +115,8 @@ spec:
                   key: {{ .hostUriConfig.key | quote }}
             - name: SMTPSERVER__PORT
               value: {{ .port | quote }}
+            - name: SMTPSERVER__EMAILSUBJECTPREFIX
```

Comment:
Can you please copy these over to platform-helm?

Reply:
can & will do 😄